storage/database/dynamodb.go

```diff
@@ -373,7 +373,7 @@ func (dynamo *dynamoDB) Close() {
 	if dynamoOpenedDBNum > 0 {
 		dynamoOpenedDBNum--
 	}
-	if dynamoOpenedDBNum == 0 {
+	if dynamoOpenedDBNum == 0 && dynamoWriteCh != nil {
```

Comment:
Is this related to this PR?

Reply:
Little bit. 
`dynamoWriteCh` is shared by all DynamoDB databases. If I didn't remove "lightchaindata" from DB names or another DB name is introduced, `dynamoWriteCh` is forced to be closed more than one time.